src/store/dashboard/spending/sagas.js

```diff
@@ -0,0 +1,15 @@
+import { put, takeLatest } from 'redux-saga/effects';
+import { FETCH_SPENDING, FETCH_SPENDING_FULFILLED } from '../../constants';
+import data from '../../../data/spending.json';
+
+export function* fetchingSpending(action, actionData = data) {
```

Comment:
maybe change actionData to spendingData?

Reply:
Done